src/Premium/Attribution.php

```diff
@@ -137,18 +146,72 @@ public function specialThanksControl( $configs ) {
 	 * @param array $controls [description]
 	 */
 	public function addControls( $controls ) {
+		global $boldgrid_theme_framework;
+		$bgtfw_version = $boldgrid_theme_framework->get_configs()['framework-version'];
+		$is_crio       = version_compare( $bgtfw_version, '2.0.0', 'ge' );
+
+		if ( $is_crio ) {
+			return $controls;
+		}
+
 		$controls = array_merge( $controls, $this->getControls() );
 
 		if ( ! get_option( 'boldgrid_reseller', false ) ) {
 			unset( $controls['reseller_control'] );
 		}
 		if ( ! $this->getLicensed() ) {
 			unset( $controls['special_thanks_control'] );
+		} else {
+			/*
+			 * Add "Hide BoldGrid Attribution" control.
+			 *
+			 * @see Crio_Premium_Customizer::add_attribution_controls
+			 *
+			 * Once, this was a privilege only for Crio Pro. When Crio was added to Inspirations, this
+			 * control was made available to all "premium" users.
+			 */
+			remove_action( 'customize_controls_print_styles', [ 'Boldgrid_Framework_Customizer_Footer', 'customize_attribution' ], 999 );
```

Comment:
I think this only applies to bgtfw v2... That being the case, it doesn't need to be here because of the above "return if Crio" - it'll never be ran